src/utils/lowercaseSwapPlugins.ts

```diff
@@ -0,0 +1,14 @@
+import { asSwapPlugin } from './config'
+
+export const getLowercaseSwapPlugins = (pluginObj): string[] => {
+  const swapPluginNamesArr: string[] = []
+  for (const [key, value] of Object.entries(pluginObj)) {
+    try {
+      asSwapPlugin(value)
```

Comment:
Is this here to throw an error if the value doesn't match the expected?

Reply:
In `serverConfig.json`, there is a `plugins` property that includes various types of plugins, including currency plugins and swap plugins. I am using the `asSwapPlugin` cleaner to only grab the swap plugins, whose values should have a type of `object`. If the plugin is not a swap plugin, it will throw a TypeError and continue onto the next one.

Reply:
Got it, your logic makes sense here.  I didn't realize the plugins would have multiple types.